.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@
 | License         | MIT
 
 <!--
- - Bug fixes must be submitted against the 1.13 branch
+ - Bug fixes must be submitted against the 1.14 branch
```

Comment:
We can also add here that security fixes are created opposite 1.13 although support is until April 2025 so I don't know if it makes sense to add this information.